.github/workflows/ci.yml

```diff
@@ -219,29 +223,55 @@ jobs:
           name: coverage-html
           path: htmlcov
 
-# FastAPI has a version constraint of pydantic<2.0.0, so we can't run tests, we expect them to break for now anyway
-#  test-fastapi:
-#    name: test fastAPI
-#    runs-on: ubuntu-latest
-#    steps:
-#    - uses: actions/checkout@v3
-#
-#    - name: set up python
-#      uses: actions/setup-python@v4
-#      with:
-#        python-version: '3.10'
-#
-#    - name: install
-#      run: |
-#        pip install -r requirements/pyproject-all.txt
-#        pip install .
-#
-#    - name: test
-#      run: make test-fastapi
+  test-fastapi:
+    # FastAPI has a version constraint of pydantic<2.0.0,
+    # so we can't run tests, we expect them to break for now anyway
+    # FIXME: drop this commend and the if-clause once that's fixed
+    if: false
```

Comment:
This seems more elegant than commenting out. Plus, it's a great opportunity to demonstrate the action's `allowed-skips` option.